algebraic-types/README.md

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
I would love to integrate all these tools in with the rest of the tools here, rather than keeping them in a separate ADT-specific sandbox. Justifications below.